.github/workflows/python-package.yml

```
@@ -7,7 +7,9 @@ on:
   push:
     branches: [ main ]
```

```suggestion
    branches:
      - main
```
just a change to rerun coverage